### PR TITLE
Add translation to Dutch for json

### PR DESCRIPTION
**Pull request by m-decoster**
I added a Dutch translation for the JSON document.

### CODE REVIEW

File: nl-nl/json.html.markdown [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+filename: learnjson.json
+contributors:
+  - ["Anna Harren", "https://github.com/iirelu"]
+  - ["Marco Scannadinari", "https://github.com/marcoms"]
+translators:
+  - ["Mathieu De Coster", "https://github.com/m-decoster"]
+lang: nl-nl
+---
+
+Aangezien JSON een extreem eenvoudig data-uitwisselingsformaat is, zal dit waarschijnlijk
```
Comment (vendethiel) [inline nl-nl/json.html.markdown]: datauitwisselingsformaat*

File: nl-nl/json.html.markdown [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+van C (`//`, `/* */`) aanvaarden. In dit voorbeeld zal alles 100% correcte JSON zijn.
+Gelukkig spreekt het meeste voor zichzelf.
+
+```json
+{
+  "key": "value",
+
+  "keys": "moeten altijd tussen dubbele aanhalingstekens staan",
+  "getallen": 0,
+  "strings": "Hellø, world. Alle unicode karakters zijn toegelaten, zo ook \"escaping\".",
```
Comment (vendethiel) [inline nl-nl/json.html.markdown]: "Unicode-"

File: nl-nl/json.html.markdown [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    "array": [0, 1, 2, 3, "Arrays kunnen eender wat bevatten.", 5],
+
+    "nog een object": {
+      "commentaar": "Hoe handig, we kunnen objecten nesten."
+    }
+  },
+
+  "dwaasheid": [
+    {
+      "bronnen van kalium": ["banenen"]
```
Comment (vendethiel) [inline nl-nl/json.html.markdown]: bananen*

File: nl-nl/json.html.markdown [hunk 4]
```diff
@@ -0,0 +1,61 @@
+---
+language: json
+filename: learnjson.json
```
Comment (vendethiel) [inline nl-nl/json.html.markdown]: please add a `-nl` suffix here.

**Comment by vendethiel** [comment]
sorry for the delay!

**Comment by m-decoster** [comment]
No problem. It should be correct now. @vendethiel

**Comment by vendethiel** [comment]
Thanks! :purple_heart: :heart: :blue_heart: :yellow_heart:

**Pull request merged**